src/main/java/com/google/codeu/data/Post.java

```diff
@@ -0,0 +1,37 @@
+package com.google.codeu.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class Post extends Article {
+    private List<Message> comments;
+
+    public Post(String authors, String header, String body) {
+        super(authors, header, body);
+        comments = new ArrayList<>();
+    }
+
+    public Post(String authors, String tags, String header, String body) {
+        super(authors, tags, header, body);
+        comments = new ArrayList<>();
+    }
+
+    public Post(UUID id, String authors, String tags, String header, String body, long timestamp) {
```

Comment:
`long` is `Long`

Reply:
`long` was used in Article Class and Message class. Therefore, for the sake of consistency, the same data type was used here.